providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/InProcessResolver.java

```diff
@@ -100,38 +106,38 @@ public void init() throws Exception {
     public void shutdown() throws InterruptedException {
         flagStore.shutdown();
         this.connected.set(false);
-        stateConsumer.accept(ProviderState.NOT_READY);
+        stateConsumer.accept(ProviderState.NOT_READY,null);
     }
 
     /**
      * Resolve a boolean flag.
      */
     public ProviderEvaluation<Boolean> booleanEvaluation(String key, Boolean defaultValue,
-            EvaluationContext ctx) {
+                                                         EvaluationContext ctx) {
```

Comment:
[nit] this formatting changes look a little bit odd to me

Reply:
sure will update